kitsune/users/jinja2/users/edit_profile.html

```diff
@@ -4,126 +4,130 @@
 {% set title = _('Edit Profile') %}
 {% set classes = 'profile' %}
 {% set active = 'edit-profile' %}
-{% set crumbs = [(profile_url(profile.user), profile.user.username), (None, title)] %}
```

Comment:
Is there a reason behind removing the breadcrumbs?

Reply:
just missed that - I need to uncomment the commented out section